src/views/Publish.vue

```diff
@@ -150,6 +159,55 @@ export default defineComponent({
       apiVersion: RequestsUtils.confAPIVersion,
       // loading indicator
       isPublishLoading: false,
+
+      // Publish History
+      columns: [
+        {
+          title: 'Full Date',
+          fieldNames: ['full_date'],
+          isSortByOriginalValue: true,
+          displayFunction: (item: any) => {
+            const newDate = new Date(item['full_date'])
+            return DateTimeUtils.isoToNowFullCuriefenseFormat(newDate)
+          },
+          isSortable: true,
+          isSearchable: true,
+          classes: 'ellipsis width-170px',
```

Comment:
No need for ellipsis, we know the maximum length of this cell - note that it has changed in `main` branch already so pull from main and now 140px should be enough

```suggestion
          classes: 'width-140px',
```

After this change please make sure the data is displayed without issues